pontoon/insights/static/js/insights.js

```diff
@@ -637,3 +746,14 @@ var Pontoon = (function (my) {
         },
     });
 })(Pontoon || {});
+
+/* Main code */
+$(function () {
+    $('body').on('click', '#insights .suggestions-age nav li', function () {
+        var items = $('.suggestions-age nav li').removeClass('active');
+        $(this).addClass('active');
+        var index = items.index(this);
+        var itemWidth = $('.suggestions-age-item').first().outerWidth();
+        $('.suggestions-age-items').css('marginLeft', -index * itemWidth);
+    });
+});
```

Comment:
```suggestion
$('#insights .suggestions-age nav li').on('click', function () {
    var items = $('.suggestions-age nav li').removeClass('active');
    $(this).addClass('active');
    var index = items.index(this);
    var itemWidth = $('.suggestions-age-item').first().outerWidth();
    // Show the selected graph view
    $('.suggestions-age-items').css('marginLeft', -index * itemWidth);
});
```

The bundle is included at the end of the page, so there's no need to add a `$()` wrapper. Also see other suggestions in the code snippet above.

Reply:
Good point. I have to keep the click event handler on the body element, though, because the Insights tab DOM is not there yet when the handler get attached.